### PR TITLE
Update `tree-equalp` to scan all `blossom-node` slots

### PR DESCRIPTION
**Pull request by karalekas**
Also [Allow pingable to be set via an initarg](https://github.com/dtqec/anatevka/commit/0e9436121961083371868b4361cd01d97e3cb63a).

### CODE REVIEW

File: tests/node.lisp [hunk 1]
```diff
@@ -261,17 +261,29 @@ Finally, all of the nodes constructed by this BLOSSOM-LET are stashed in the pla
...
+                  (equalp left-value right-value))))
+             (blossom-slots (value)
+               (initialize-and-return ((slots nil))
+                 (block quick-exit
+                   (dolist (superclass (closer-mop:class-precedence-list (class-of value)))
+                     (dolist (slot-definition (closer-mop:class-direct-slots superclass))
+                       (push (closer-mop:slot-definition-name slot-definition) slots))
+                     (when (equalp superclass (closer-mop:ensure-class
+                                               'anatevka:blossom-node))
+                       (return-from quick-exit)))))))
```
Comment (ecpeterson) [inline tests/node.lisp:273]: I know I wrote this, but I’m sure this isn’t idiomatic and I can’t remember what the right thing to do is. Some form provides an “implicit tagbody” which one can escape using a bare `(return)`, but…

**Review by ecpeterson — APPROVED** [review]
If you can figure out how you’d my one (self-)complaint, please do. If not, we’ll live.

File: tests/node.lisp (hunk 1 above)
Comment (ecpeterson) [reply tests/node.lisp:273]: Maybe prog, from https://stackoverflow.com/questions/4394942/which-standard-common-lisp-macros-special-forms-establish-implicit-blocks-named

Comment (karalekas) [reply tests/node.lisp]: took the `block nil` suggestion

**Pull request merged**